### DISCUSSION TITLE
How to use react-markdown in a standalone HTML via CDN

### DISCUSSION THREAD

**Discussion by rajiff** [remarkjs/remark#821] [Q&A]
Want to use `react-markdown` for a small standalone HTML page (for a Documentation project), as my content will be in markdown, want to render the markdown in this page

Looking for an example on including `react-markdown` via CDN and use it in my below standalone HTML page

Any help welcome

```html
<!DOCTYPE html>
<html>

<head>
</head>

<body>
  <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>

  <!-- Load React. -->
  <script crossorigin src="https://unpkg.com/react@17/umd/react.production.min.js"></script>
  <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.production.min.js"></script>
  <script src="https://unpkg.com/@material-ui/core@latest/umd/material-ui.production.min.js" crossorigin></script>

  <!-- Not working or don't know correctly how to use it -->
  <script src="https://cdn.jsdelivr.net/npm/react-markdown@7.0.0/index.min.js"></script>

  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" />
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />

  <div id="app"></div>

  <script type="text/babel">
    const { Typography, Icon } = MaterialUI;

    function SampleMarkApp() {
        return (
        <div>
            <Typography variant="h4">{"How to use markdown in standalone HTML?"}</Typography>
            <ReactMarkdown>{"# Hello, *world*!"}</ReactMarkdown>
        </div>
        )
    }
    ReactDOM.render(<SampleMarkApp />, document.querySelector('#app'));
</script>
</body>

</html>
```

Get this error

![image](https://user-images.githubusercontent.com/1080415/130812949-a3eb031a-0903-4ac8-ab44-f34525264da8.png)

Can try it here https://jsbin.com/weqesip/edit?html,output

**Comment by wooorm** [comment]
Skypack is a pretty good CDN: https://www.skypack.dev/view/react-markdown

**Reply by wooorm** [reply]
The same can be done with `jsdelivr` which you are using: `import` it in a `<script type="module">`

**Reply by rajiff** [reply]
@wooorm thanks skypack looks good

The syntax from skypack can it be used in a standalone HTML, i tried, but not much luck it says ReactMarkdown not found

```
<script type="module">
  import reactMarkdown from 'https://cdn.skypack.dev/react-markdown';
</script>
````

**Reply by wooorm** [reply]
I raised an issue for you: https://github.com/skypackjs/skypack-cdn/issues/210.

**Comment by Murderlon** [comment]
You just added the script, you didn't render any content with `react-markdown`.

From the README: 
```js
const markdown = `Just a link: https://reactjs.com.`

ReactDom.render(
  <ReactMarkdown children={markdown} remarkPlugins={[remarkGfm]} />, // this part
  document.body
)
```

You would just pass your markdown, not the entire React tree.

--- 

Alternatively, if you like React and markdown, you could save yourself the setup and use something like https://docusaurus.io/

**Reply by rajiff** [reply]
Sorry had missed that from the HTML but i had tried (now updated)

Tried your way too but not sure something is missing
